src/modules/api.js

```diff
@@ -0,0 +1,9 @@
+const apiUrl = "https://api.tvmaze.com/shows/1/seasons";
```

Comment:
This API seems static. It's better to use a dynamic API, changing with shows' ID, like shows/:id 
Then it will get the ID of the show and fetch the related information.